multi_physics/QED/kokkos_example/kokkos_example_quantum_synchrotron.cpp

```diff
@@ -0,0 +1,311 @@
+#include "kokkos_example_commons.hpp"
+
+// QUANTUM SYNCHROTRON EMISSION
+
+//Parameters of the test case
+const unsigned int how_many_particles = 20'000'000;
+const unsigned int how_many_repetitions = 1;
+const double dt_test= 1e-18;
+const double table_chi_min = 0.01;
+const double table_frac_min = 1.0e-12;
+const double table_chi_max = 1000.0;
+const int table_chi_size = 128;
+const int table_frac_size = 128;
+const int random_seed = 22051988;
+const double E_min = -0.01*Es;
+const double E_max = 0.01*Es;
+const double B_min = E_min/pxr::light_speed<>;
+const double B_max = E_max/pxr::light_speed<>;
+const double P_min = -100*mec<>;
+const double P_max = 100*mec<>;
+//__________________________________________________
+
+/**
+* Generates the dN/dt lookup table
+*
+* @tparam Real the floating point type to be used
+* @tparam Vector the vector type to be used
+* @param[in] chi_min the minimum chi parameter
+* @param[in] chi_max the maximum chi parameter
+* @param[in] chi_size the size of the lookup table along the chi axis
+* @return the lookup table
+*/
+template <typename Real, typename Vector>
+auto generate_dndt_table(Real chi_min, Real chi_max, int chi_size)
+{
+    std::cout << "Preparing dndt table [" << get_type_name<Real>()
+        << ", " << chi_size <<"]...\n";
+    std::cout.flush();
+
+    pxr_qs::dndt_lookup_table_params<Real> qs_params{chi_min, chi_max, chi_size};
+
+	auto table = pxr_qs::dndt_lookup_table<
+        Real, Vector>{qs_params};
+
+    table.generate();
+
+    return table;
+}
+
+/**
+* Generates the photon emission lookup table
+*
+* @tparam Real the floating point type to be used
+* @tparam Vector the vector type to be used
+* @param[in] chi_min the minimum chi parameter
+* @param[in] chi_max the maximum chi parameter
+* @param[in] frac_min the mimum value for the frac parameter
+* @param[in] chi_size the size of the lookup table along the chi axis
+* @param[in] frac_size the size of the lookup table along the frac axis
+* @return the lookup table
+*/
+template <typename Real, typename Vector>
+auto generate_photon_emission_table(
+    Real chi_min, Real chi_max, Real frac_min, int chi_size, int frac_size)
```

Comment:
```suggestion
    const Real chi_min, const Real chi_max, const Real frac_min, const int chi_size, const int frac_size)
```
Maybe needs another line here. These could be references as well, but it's really nitpicking at this point.